test/bloomfilter.js

```diff
@@ -2,16 +2,76 @@
 
 var chai = require('chai');
 
+var assert = require('assert');
+var bitcore = require('bitcore');
 var Data = require('./data/messages');
 var P2P = require('../');
 var BloomFilter = P2P.BloomFilter;
 
+// convert a hex string to a bytes buffer
+function ParseHex(str) {
```

Comment:
I think this exists in a few `test/*` files, should probably be put into a `test/util.js` eventually.